canonical/loki-k8s-operator
# Fix api call

## Issue
Fixes #205


## Solution
Insert path_prefix in to the api call url.


## Testing Instructions
* Deploy Loki
* Relate to Traefik
* Relate to a logging client
* See if Loki is up


## Release Notes
Fixed a bug where Loki-k8s fails when used with Traefik-k8s
